repo/packages/J/jenkins/0/marathon.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "jenkins",
-  "instances": 1,
-  "mem": 1500,
+  "instances": 1.5,
+  "mem": 2048,
   "env": {
     "JAVA_OPTS": "-Xmx300m"
```

Comment:
hmm you're right. I might not be initializing the memory limit correctly. That could explain the crashes.